monero-outreach-docs/translations/es/PSA/files/mining/mine_support_network_es.md

```diff
@@ -0,0 +1,27 @@
+# Minar Monero para apoyar la red
+
+Los mineros de Monero utilizan hardware informático para ejecutar una variante del algoritmo Cryptonight y competir por una recompensa asociado al siguiente bloque en la blockchain. La minería de Monero es reconocida por su resistencia a los ASIC. Esto quiere decir que es más accesible y amigable minar con CPU (unidad central de proceso) y hardware GPU (unidad de procesamiento gráfico), pero no garantiza la configuración que realizan los individuos y las compañias a los poderosos mineros ASICs y los FPGAs para minar Monero. Los desarrolladores de Monero han establecido una actualización rutinaria sobre la red, que tiene lugar cada 6 meses que ayuda a combatir en contra de la minería de Monero a través de los ASICs y los FPGAs y evitar que estos puedan obtener el control de la red a través del hashrate (poder de procesamiento).
+
+Sin embargo, se ha asumido que el valor intrínseco o la rentabilidad general de la minería de Monero empieza a ser notable, más individuos y empresas intentarán configurar los ASICs y los FPGAs para tener una ventaja comparativa y llevarse las recompensas de la minería en contra de los mineros CPU/GPU con un hashrate menor. Algunos miembros de la comunidad les preocupa el riesgo sobre un ataque del 51% en Monero de parte de los ASICs y los FPGAs con su alto poder de procesamiento hash que sus máquinas pueden ofrecer.
+
+Como resultado de esto, hay un *llamado a la acción* para defender y asegurar la red de Monero *para minar con cualquier medio posible que se tenga a disposición*. Te animamos para que te conviertas en un participante activo en la red de Monero y ayudes a asegurarla. 
+
+¡Únete a la lucha para proteger el efectivo privado! He aquí unas opciones prácticas que podrías considerar para ayudar a proteger la red:
+
+1. **Minería en solitario (Solo Mining)-** Esto es visto como la opción de la lotería en la minería. Si prendes el minador en el CPU o GPU de tu computadora para minar Monero, estarás apostando que ganarás algún bloque frente a todos los otros mineros en la red. Recibirás el total de la recompensa de ese bloque, pero las probabilidades de que esto suceda realmente son bajas. Puedes minar en solitario desde los monederos oficiales GUI y CLI de Monero ([https://www.getmonero.org/es/resources/user-guides/solo_mine_GUI.html](https://www.getmonero.org/es/resources/user-guides/solo_mine_GUI.html)).
```

Comment:
- _...si **enciendes** el minero..._ _Minero_ it's a suggestion, it sounds more normal to me, but _enciendes_ is more neutral.
- _...de que esto suceda son realmente bajas._

Reply:
oh I think I missed this

Reply:
done :)